spec/integration/v1/contacts_spec.rb

```diff
@@ -10,52 +10,16 @@
     @dk_token = doorkeeper_token
   end
 
-  path '/api/v1/contacts' do
-    get 'List all Contacts' do
+  path '/api/v1/contacts/' do
+    get 'Return 404 error for index call' do
```

Comment:
Then you can just remove these specs - because the route doesn't even exist! :)

Reply:
Done